worlds/sm/Options.py

```diff
@@ -52,6 +52,14 @@ class DeathLink(Choice):
     alias_true = 1
     default = 0
 
+class RemoteItems(Toggle):
+    """Indicates you get items sent from your own world. This allows coop play of a seed."""
+    display_name = "Remote Items"  
+
+class RemoteStartInventory(Toggle):
```

Comment:
I don't think there's a good reason to do this? If you have the ability to bake them in, that's just better, right?

Reply:
Maybe to have the game behave uniformly when Remote Items is used? To be honest, I did this more based on the sight of the bit flag that is now used for item handling, to be as flexible as possible. On the other hand, I understand this might cluther the available options seen by players for not much. So, just tell me if you'd prefer me to remove this.

Reply:
It's probably best to remove it. I struggle to find an advantage to it. But I can find some issues, like player confusion.